src/Zynga/Framework/Cache/V2/Config/LocalMemcache/PgResultSetTest/Dev.hh

```diff
@@ -31,4 +31,8 @@ class Dev extends LocalMemcacheBase {
 
   }
 
+  public function cacheAllowsTTLOverride(): bool {
```

Comment:
lets also add update the actual pgdata configs in the db-config repo and link that PR in here